xarray/core/dataarray.py

```diff
@@ -1228,6 +1228,97 @@ def fillna(self, value):
         out = ops.fillna(self, value)
         return out
 
+    def interpolate_na(self, dim=None, method='linear', limit=None,
```

Comment:
Probably too late to be helpful - but are we sure about the name here? We don't generally add `_na` onto methods (`bfill_na`?), and pandas is `interpolate` only

Reply:
See comment from @shoyer above: https://github.com/pydata/xarray/pull/1640#discussion_r145888076